modules/custom/d_commerce/config/optional/block.block.droopler_subtheme_user_account_menu.yml

```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - system.menu.account
+  module:
+    - system
+  theme:
+    - droopler_subtheme
```

Comment:
Ten config powinien być dla droopler_theme raczej nie? A w kodzie wygenerowany dla subtheme?

Reply:
Tak myślę że skoro Drooplera nie można dodać do już zainstalowanego Drupala tylko profil instaluje Drupala razem z Drooplerem to zamiast tworzyć ten config dla subtheme dynamicznie lepiej go dodać tutaj niech się odrazu zaimportuje podczas instalacji ?

Reply:
Tylko, że nie kazdy musi mieć droopler_subtheme... Jak ktoś nie ma, to ma tylko droopler_theme.
A moze ktoś od razu zaczynać ze swoim customowym np. my_custom_droopler_subtheme.

Reply:
no tak, ten subtheme to jest cos czego unikamy w configach